### PR TITLE
add storage to filelog receiver for extraFileLog

### PR DESCRIPTION
**Pull request by omrozowicz-splunk**
This PR adds a `storage` field for `extraFileLog` if it wasn't provided in the `values.yaml`.
Fix for: https://github.com/signalfx/splunk-otel-collector-chart/issues/692

### CODE REVIEW

**Review by atoulme — APPROVED** [review]
Changes look good, example has the file_storage storage field set.

**Review by jvoravong — APPROVED** [review]

File: helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl [hunk 1]
```diff
@@ -405,7 +405,14 @@ receivers:
   {{- end }}
 
   {{- if .Values.logsCollection.extraFileLogs }}
-  {{- toYaml .Values.logsCollection.extraFileLogs | nindent 2 }}
+  {{- $extraFileLogsList := .Values.logsCollection.extraFileLogs }}
+  {{- range $extraFileLogKey, $extraFileLogValue := $extraFileLogsList }}
+  {{- printf "%s:" $extraFileLogKey | nindent 2 }}
+  {{- if not $extraFileLogValue.storage }}
+    storage: file_storage
```
Comment (dmitryax) [inline helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl:412]: I'm curios how this works with 2 spaces indent. I believe it should be 4, no?

**Review by dmitryax — COMMENTED** [review]
@omrozowicz-splunk Please rebase and run `make render`

File: helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl (hunk 1 above)
Comment (omrozowicz-splunk) [reply helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl:412]: Actually there are 4 spaces before `storage: file_storage`. I matched the indentation of the other elements of the filelog receiver.

**Review by dmitryax — APPROVED** [review]

**Comment by omrozowicz-splunk** [comment]
Added another commit to fix example file, as I thought it's such a small change it doesn't require testing (and forgot using mounts 🤯 ), now I simplified it and tested on real env. I deleted `another storage` case as well as it requires a ton of additional config and I'm not sure if it's even useful for anyone.

**Pull request merged**